.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -9,16 +9,31 @@ assignees: ''
 
 ---
 
+I have downloaded the latest cli from: https://checkstyle.org/cmdline.html#Download_and_Run
+I have executed the cli and showing it below, as cli describes the problem better than 1,000 words
+
+**How it works Now:**
+
+```bash
+/var/tmp $ javac YOUR_FILE.java
+#[[MAKE SURE THERE IS SUCCESSFUL COMPILATION]]
+
+/var/tmp $ cat config.xml
+#[[PLACE YOUR OUTPUT HERE]]
+
+/var/tmp $ cat YOUR_FILE.java
+#[[PLACE YOU OUTPUT HERE]]
+
+/var/tmp $ RUN_LOCALE="-Duser.language=en -Duser.country=US"
+/var/tmp $ java $RUN_LOCALE -jar checkstyle-X.XX-all.jar -c config.xml YOUR_FILE.java
+#[[PLACE YOUR OUTPUT HERE]]
+```
+
 **Is your feature request related to a problem? Please describe.**
 A clear and concise description of what the problem is.
 
 **Describe the solution you'd like**
 A clear and detailed description of what you want to happen.
```

Comment:
Shouldn't solution have expected CLI output for "fix"?

Reply:
I have added it that they could provide the cli suggested output for their fix.